src/AtomType.jl

```diff
@@ -0,0 +1,25 @@
+struct AtomType
```

Comment:
My reasoning was that we might want Atom (representing an atom as participating in the computation, ie with a position etc.) whereas this is the abstract struct containing the information about a type of atoms.

Reply:
Ah I see, good point. Still I think it is a duplicate to call a type `BlaType`. To stay in the chemistry language your `AtomType` would essentially be a model for an `Element`, actually. Otherwise I guess we could just leave it as `Species` as it was. Renaming the list of species to `atoms` is a good step, I think.

Reply:
So should we just rename it to `Element`?